CDCgov/prime-reportstream
# Experience/9722/programmers guide style updates

This PR ...

Fix vertical spacing between nav and page content/nav on API Programmer's Guide page
Fixed Admin button hover styling.
Test Steps:

Log into RS

1. Select Resources -> API Programmers Guide
2. Notice that there is no space b/w the nav and the header
![Screenshot 2023-07-05 at 4 06 34 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/1b07b6f1-d204-4a5d-acda-f040ae3f0e22)


### Testing


## Linked Issues
- Fixes #9722
